.travis.yml

```diff
@@ -33,5 +34,5 @@ notifications:
   slack:
     secure: eX3VgtKQ48rzKGdXpIZZVYINffI2wbqhqoJFUxHO1Zku6tXzbt0R+r3NVHo37wBID63bkXjYqjZuv+JRUtvf/XO51+QkTqCCZ6iC3lr6IEqRpzxXsmNYf+QPhuN2kvVTSyycHFXdwQLL7sCniFSmEcLdu6xMpobI4PUwi2OvHj4=
 before_install:
-- openssl aes-256-cbc -K $encrypted_6a9d32f3e0bd_key -iv $encrypted_6a9d32f3e0bd_iv
-  -in ci.enc -out testing/ci -d
+  - if [ -z "$ARE_SECRETS_AVAILABLE" ]; then MANTL_CI_FORK_CHECK=1; else MANTL_CI_FORK_CHECK=0; fi
```

Comment:
So how's that work? The settings are only populated if the secrets are available?